lib/plugins/physics.js

```diff
@@ -194,7 +195,7 @@ function inject (bot) {
 
   let lookAtCb = noop
   function checkYaw () {
-    if (Math.abs(((lastSentYaw % PI_2) - bot.entity.yaw) % PI_2) < 0.001) {
+    if (Math.abs(deltaYaw) < 0.001) {
```

Comment:
can you use a function to compute this ?
Defining a variable at the top like you did is confusing and potentially wrong.